platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -42,6 +42,7 @@
 import com.swirlds.platform.internal.EventImpl;
 import com.swirlds.platform.metrics.ConsensusMetrics;
 import com.swirlds.platform.system.address.AddressBook;
+import com.swirlds.platform.util.MarkerFileWriter;
```

Comment:
Adding this as a comment so that you have to press resolve before merging. Don't forget to set the milestone prior to merging (which milestone will depend on when it merges, and when the 0.49 branch is cut).